Assignment1/BasicVersion/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"bufio"
+	"os"
+	"strings"
+	"errors"
+	"strconv"
+	)
+
+func main(){
+	scanner := bufio.NewScanner(os.Stdin)
+
+	fmt.Print("> ")
+	for scanner.Scan(){
+		text := scanner.Text()
+		
+		if err := evalExpression(text); err != nil {
+			fmt.Printf("Error!!! %v\n", err)
+		}
+
+		fmt.Println("> ")
+	}
+}
+
+func evalExpression(text string) error {
+	array_s := strings.Split(text, " ")
```

Comment:
Bạn có thể sử dụng strings.Fields để tách được `slice of words`
Tên biến nên đặt ngắn gọn, rỏ nghĩa, camelcase